docs/databases/matrix.md

```diff
@@ -58,11 +59,29 @@ async def skill_func(opsdroid, config, message):
 	...
 ```
 
-## Encryption
+```eval_rst
+.. autofunction:: opsdroid.database.matrix.memory_in_event_room
+```
+
+## How it Works
 
-In encrypted Matrix rooms, state events (used by the database) are not encrypted. This means that anything put into the matrix database in an encrypted room would not be encrypted. To prevent this the matrix database send the **values** you put into your database into the room as regular events, which are encrypted, and then references these events from the database (which is still a state event).
+### State Events and State Keys
+
+Matrix state events have a type, which for the opsdroid matrix database is `dev.opsdroid.database`, a "state key" which distinguishes state events with the same type, and content which is a json object.
```

Comment:
Honestly I struggle to parse this sentence a little but that may just be because I'm tired.